docs/0.4.0-alpha/index.json

```diff
@@ -4,6 +4,10 @@
             "path": "getting_started/getting_started",
             "title": "Getting Started",
             "docs": [
+                {
+                    "path": "getting_started/migration_guide",
```

Comment:
Consider reordering these pages; you can't do anything until you've installed Amber anyway, and many users (for whom 0.4.0 will be their first version) will care more about basic usage than migration:
* Installation
* Usage
* Migration Guide